### ISSUE TITLE
Blynk Edgent Provisioning Issue

### ISSUE THREAD

**Issue by hms-11** [blynkkk/blynk-library#575]
Blynk library version: [1.1.0]
IDE: [Arduino]
IDE version: [2.0.3]
Board type: [ESP32]
Additional modules: [...]

### Scenario, steps to reproduce
[When provisioning a new board with an Edgent based sketch, board will not provision correctly and will give a 701 error code in the serial monitor before going back into AP mode and awaiting configuration. If I attempt to provision on an older phone (Google Pixel 5) the process works correctly, if I use my new phone (Google Pixel 7) the process fails as described. Both phones are running the same version of android and the same version of the Blynk App.
See this thread in the Blynk Community for more details: https://community.blynk.cc/t/blynkedgent-based-sketch-cannot-connect-to-wifi/63923 .]

### Expected Result
[After supplying correct wifi info, board will reboot, not connect and eventually re-boot again and go back into provisioning mode.]

### Actual Result
[Board fails provisioning and returns to AP mode awaiting credentials if using a new Pixel 7, if using my previous phone the process works fine.]

**Comment by John93-Blynk** [comment]
I would suggest that you go to the Blynk forum and create a new thread there.
https://community.blynk.cc/

**Comment by hms-11** [comment]
Hey @John93-Blynk,
I started over there at the Blynk forum and after Pete and I had exhausted all easily solvable issues we decided I should post an issue. 
However, that being said, The BlynkAndroidDev user, whom I presume is a developer for Bynks Android app said he was aware of the issue and is planning on releasing a fix today. I'll verify the fix sometime this week and update the issue if it is solved. 

Thanks!
